boards/nucleo-f334r8/include/periph_conf.h

```diff
@@ -89,7 +115,11 @@ static const uart_conf_t uart_config[] = {
         .rx_af      = GPIO_AF7,
         .tx_af      = GPIO_AF7,
         .bus        = APB1,
-        .irqn       = USART3_IRQn
+        .irqn       = USART3_IRQn,
+#ifdef MODULE_PERIPH_DMA
+        .dma        = 0,
```

Comment:
This dma channel is shared with SPI1 RX so that won't work if both are used and DMA is enabled. (I have no idea how to solve this in the current RIOT state).

Reply:
Should we disable USART3 when both MODULE_PERIPH_SPI and MODULE_PERIPH_DMA are enabled ?

Reply:
> Should we disable USART3 when both MODULE_PERIPH_SPI and MODULE_PERIPH_DMA are enabled ?

Yes, I think this is the best approach.

Reply:
Like this ?